tools/gen-intrinsics-specs.py

```diff
@@ -741,8 +819,8 @@ def process_db(db, classification_db):
     body = ""
     for k, v in filtered.items():
         body += "\n" + \
-            recurse_print_to_rst((k, v), rst_header_levels,
-                                 headers=table_header, tablefmt="grid")
+            recurse_print_to_rst((k, v), rst_header_levels if table_format == "grid" else md_header_levels,
```

Comment:
Yeah, this `rst_header_levels if table_format == "grid" else md_header_levels` would simplify a lot with the dict mentioned in my previous comment.